KTLOSNaxxLoot.json

```diff
@@ -216,7 +216,7 @@
 			"wowID":"23030"
 		},
 		"Noth's Frigid Heart":{
-			"prio":["Peku","LC","EP/GP"],
+			"prio":["Peku2","Nisio(2)","LC","EP/GP"],
```

Comment:
Can I has first? Using fire runed grimore...

Reply:
Only if I get the priest ring before you :P

Reply:
Ring of faith?
Sure... but I'm not even on it lol

Reply:
Done